homeassistant/components/bang_olufsen/media_player.py

```diff
@@ -461,7 +461,7 @@ def source(self) -> str | None:
         ):
             if (
                 len(self._playback_metadata.art) == 0
-                and self._source_change.id == BangOlufsenSource.BLUETOOTH.id
+                and self._source_change.id == BangOlufsenSource.CHROMECAST.id
             ):
```

Comment:
There is no need for the nested `if`
I think it's better to combine the four conditions.